Intermittent integration test failure: test_create_new_experiment_timeout_remote_settings

Seen here https://app.circleci.com/pipelines/github/mozilla/experimenter/12735/workflows/62bd720a-05d9-4697-944e-9fae692b3641/jobs/38409

```
_________ test_create_new_experiment_timeout_remote_settings[DESKTOP] __________

selenium = <selenium.webdriver.firefox.webdriver.WebDriver (session="c4703e30-8bf5-4d5b-a429-4a46e7c2c7ef")>
create_experiment = <function create_experiment.<locals>._create_experiment at 0x7f7859e8a940>

    @pytest.mark.run_once
    def test_create_new_experiment_timeout_remote_settings(
        selenium,
        create_experiment,
    ):
        summary = create_experiment(selenium)
        summary.launch_and_approve()
>       summary.wait_for_timeout_alert()

nimbus/test_remote_settings.py:40: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
nimbus/pages/experimenter/summary.py:95: in wait_for_timeout_alert
    self.wait_with_refresh(
nimbus/pages/base.py:27: in wait_with_refresh
    self.wait.until(_wait_for_it, message=message)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <selenium.webdriver.support.wait.WebDriverWait (session="c4703e30-8bf5-4d5b-a429-4a46e7c2c7ef")>
method = <function Base.wait_with_refresh.<locals>._wait_for_it at 0x7f7859e26160>
message = 'Summary Page: Unable to find timeout alert'

    def until(self, method, message=''):
        """Calls the method provided with the driver as an argument until the \
        return value is not False."""
        screen = None
        stacktrace = None
    
        end_time = time.time() + self._timeout
        while True:
            try:
                value = method(self._driver)
                if value:
                    return value
            except self._ignored_exceptions as exc:
                screen = getattr(exc, 'screen', None)
                stacktrace = getattr(exc, 'stacktrace', None)
            time.sleep(self._poll)
            if time.time() > end_time:
                break
>       raise TimeoutException(message, screen, stacktrace)
E       selenium.common.exceptions.TimeoutException: Message: Summary Page: Unable to find timeout alert

.tox/integration-test-nimbus/lib/python3.8/site-packages/selenium/webdriver/support/wait.py:80: TimeoutException
```

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/EXP-2496)

--- a/app/tests/integration/nimbus/pages/base.py
+++ b/app/tests/integration/nimbus/pages/base.py
@@ -7,7 +7,7 @@ class Base(Page):
     """Base page."""
 
     def __init__(self, selenium, base_url, **kwargs):
-        super().__init__(selenium, base_url, timeout=60, **kwargs)
+        super().__init__(selenium, base_url, timeout=80, **kwargs)
 
     def wait_for_page_to_load(self):
         self.wait.until(EC.presence_of_element_located(self._page_wait_locator))

